miden-lib/asm/sat/internal/constants.masm

```diff
@@ -20,9 +20,9 @@ const.MAX_NUM_CREATED_NOTES=4096
 # zeros for a regular account (2^23).
 const.REGULAR_ACCOUNT_SEED_DIGEST_MODULUS=8388608
 
-# Specifies a modulus multiplier used to asses if an account seed digest has the required number of 
+# Specifies a modulus shift used to asses if an account seed digest has the required number of 
 # trailing zeros for a faucet account (2^8).
-const.FAUCET_SEED_DIGEST_EXTRA_MODULUS_MULTIPLIER=256
+const.FAUCET_SEED_DIGEST_MODULUS_SHIFT=8
```

Comment:
If we go with the approach suggested in the previous comment, this would become something like `FAUCET_ACCOUNT_SEED_DIGEST_MODULUS`.